IdentityCore/src/cache/mac/MSIDMacTokenCache.m

```diff
@@ -22,39 +22,448 @@
 // THE SOFTWARE.
 
 #import "MSIDMacTokenCache.h"
+#import "MSIDToken.h"
+#import "MSIDTokenCacheKey.h"
+#import "MSIDTokenSerializer.h"
+#import "MSIDUserInformation.h"
+
+#define CURRENT_WRAPPER_CACHE_VERSION 1.0
+
+#define RETURN_ERROR_IF_CONDITION_FALSE(_cond, _code, _details) { \
```

Comment:
nit: I'd try to avoid macroses like that going forward and I'd prefer a method or function instead. Please let me know what do you think.

Reply:
Let's add an issue for this one and discuss all together, no need to change anything now.

Reply:
agree

Reply:
added #56